website/server/libs/inbox/index.js

```diff
@@ -141,6 +142,7 @@ export async function listConversations (owner) {
   const conversations = conversationsList.map(res => ({
     uuid: res._id,
     ...res,
+    backer: Array.isArray(res.npc) && res.npc.length > 0 ? { npc: res.npc[0] } : { npc: null },
```

Comment:
@negue could you give a look at this part since you worked the most on it? thanks!